src/components/common/NameRequestInfo.vue

```diff
@@ -23,7 +23,7 @@
               <li class="mt-4"><strong>Entity Type:</strong> {{ getEntityTypeDescription }}</li>
               <li><strong>Request Type:</strong> {{ requestType }}</li>
               <li><strong>Expiry Date:</strong> {{ expirationDate }}</li>
-              <li><strong>Status:</strong> {{ state }}</li>
+              <li><strong>Status:</strong> {{ Capitalize(state) }}</li>
```

Comment:
I don't think there was a bug ticket for this, but it should be Title Case (eg, Conditional).

Reply:
Sample from Filings UI:

![image](https://user-images.githubusercontent.com/10002889/204681850-7fb5aa75-5b73-41ec-9520-415162e3d7c8.png)

Previous code in Create UI:

![image](https://user-images.githubusercontent.com/10002889/204681914-8aa33aa1-171d-4883-b59d-e2acea7b806f.png)